### PR TITLE
update(getting started): fill out the todo, fix first page

### PR DESCRIPTION
**Pull request by NathalieC**
* remove reference to overview
* fill all cards descriptions

### CODE REVIEW

File: modules/ROOT/pages/getting-started-index.adoc [hunk 1]
```diff
@@ -4,54 +4,57 @@
 This is the fastest way to learn, try, and master all concepts of a Living Application.
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]:
```suggestion
To get started with Bonita, learn how to create your first complete living application.
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 2]
```diff
@@ -4,54 +4,57 @@
...
 
 [.card-section]
-== The 9-step journey
+== Prerequisite
 
 [.card.card-index]
 --
-xref:xref:tutorial-overview.adoc[[.card-title]#Overview# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:bonita-studio-download-installation.adoc[[.card-title]#Bonita Studio Installation# [.card-body.card-content-overflow]#pass:q[Bonita's main tool to create the main elements of all your projects]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]: You use 'main' twice in a short sentence, it's better to use a synonym :)

```suggestion
xref:bonita-studio-download-installation.adoc[[.card-title]#Bonita Studio Installation# [.card-body.card-content-overflow]#pass:q[Bonita's principal tool to create the main elements of all your projects]#]
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 3]
```diff
@@ -4,54 +4,57 @@
...
 [.card.card-index]
 --
-xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation, "most of the time", starts with a process]#]
 --
 
 [.card.card-index]
 --
-xref:define-business-data-model.adoc[[.card-title]#Define a Business Data Model (BDM)# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:define-business-data-model.adoc[[.card-title]#Define a Business Data Model (BDM)# [.card-body.card-content-overflow]#pass:q[The business data will be handled by the process. First, you need to create the data model]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]: This sentence is too long, the last words are cut in the card.

```suggestion
xref:define-business-data-model.adoc[[.card-title]#Define a Business Data Model (BDM)# [.card-body.card-content-overflow]#pass:q[The data model represents the business data handled by processes.]#]
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 4]
```diff
@@ -4,54 +4,57 @@
...
 --
 
 [.card.card-index]
 --
-xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[This is the step to link the business data to the process]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]:
```suggestion
xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[Learn hot to link business data and processes]#]
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 5]
```diff
@@ -4,54 +4,57 @@
...
 --
 
 [.card.card-index]
 --
-xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[This step defines how the UI and the process interact]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]:
```suggestion
xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[Contracts are the interfaces allowing processes and UI's to interact]#]
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 6]
```diff
@@ -4,54 +4,57 @@
...
 --
 
 [.card.card-index]
 --
-xref:create-web-user-interfaces.adoc[[.card-title]#Create forms# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:create-web-user-interfaces.adoc[[.card-title]#Create forms# [.card-body.card-content-overflow]#pass:q[The forms use the data to feed the process, through the contracts]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]:
```suggestion
xref:create-web-user-interfaces.adoc[[.card-title]#Create forms# [.card-body.card-content-overflow]#pass:q[Forms use data to feed processes through contracts]#]
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 7]
```diff
@@ -4,54 +4,57 @@
...
 [.card.card-index]
 --
-xref:configure-email-connector.adoc[[.card-title]#Configure an email connector# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:configure-email-connector.adoc[[.card-title]#Configure an email connector# [.card-body.card-content-overflow]#pass:q[Most of the processes need email notifications, right?]#]
 --
 
 [.card.card-index]
 --
-xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[Unlike the form, an application page displays more than the information of a human task]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]: This sentence is too long for the card.

```suggestion
xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[An application page is not bounded to a human task, unlike a form]#]
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 8]
```diff
@@ -4,54 +4,57 @@
...
 --
 
 [.card.card-index]
 --
-xref:create-application.adoc[[.card-title]#Create an application# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:create-application.adoc[[.card-title]#Create an application# [.card-body.card-content-overflow]#pass:q[And this is where everything is put together to build a rich and valuable end-user experience]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]: This sentence is too long for the card.

```suggestion
xref:create-application.adoc[[.card-title]#Create an application# [.card-body.card-content-overflow]#pass:q[This is where everything is put together to build a rich, valuable end-user experience]#]
```

File: modules/ROOT/pages/tutorial-overview.adoc [hunk 9]
```diff
@@ -15,11 +13,12 @@ If you have any problems while working through this tutorial, you can https://co
 
 == Before starting
 
-Make sure to check out the page that tells you everything about xref:what-is-bonita.adoc[What Bonita is].
+. Make sure to check out the page that tells you everything about xref:what-is-bonita.adoc[What Bonita is]. +
```
Comment (alachambre) [inline modules/ROOT/pages/tutorial-overview.adoc]: Which page(s) ?? A link please :)

**Review by alachambre — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/tutorial-overview.adoc (hunk 9 above)
Comment (NathalieC) [inline modules/ROOT/pages/tutorial-overview.adoc]:
```suggestion
. Make sure to xref:what-is-bonita.adoc[check this chapter], that tells you everything about what Bonita is. +
```

Comment (NathalieC) [reply modules/ROOT/pages/tutorial-overview.adoc]: Made the link clearer

File: modules/ROOT/pages/getting-started-index.adoc (hunk 7 above)
Comment (NathalieC) [reply modules/ROOT/pages/getting-started-index.adoc]: je rectifie l'anglais, je revois la phrase et je commit

File: modules/ROOT/pages/getting-started-index.adoc [hunk 10]
```diff
@@ -1,57 +1,60 @@
...
 [.card.card-index]
 --
-xref:configure-email-connector.adoc[[.card-title]#Configure an email connector# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:configure-email-connector.adoc[[.card-title]#Configure an email connector# [.card-body.card-content-overflow]#pass:q[Most of the processes need email notifications, right?]#]
 --
 
 [.card.card-index]
 --
-xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[Unlike the form, an application page displays more than the information of a human task]#]
```
Comment (NathalieC) [inline modules/ROOT/pages/getting-started-index.adoc]:
```suggestion
xref:design-application-page.adoc[[.card-title]#Design an application page# [.card-body.card-content-overflow]#pass:q[Application pages information is more generic than human tasks information]#]
```

File: modules/ROOT/pages/getting-started-index.adoc (hunk 5 above)
Comment (NathalieC) [reply modules/ROOT/pages/getting-started-index.adoc]: Not UI's. Je commit avec ce petit ajustement

File: modules/ROOT/pages/getting-started-index.adoc [hunk 11]
```diff
@@ -1,57 +1,60 @@
...
 [.card.card-index]
 --
-xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:declare-business-variables.adoc[[.card-title]#Declare business variables# [.card-body.card-content-overflow]#pass:q[Learn hot to link business data and processes]#]
 --
 
 [.card.card-index]
 --
-xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[This step defines how the UI and the process interact]#]
```
Comment (NathalieC) [inline modules/ROOT/pages/getting-started-index.adoc]:
```suggestion
xref:declare-contracts.adoc[[.card-title]#Declare contracts# [.card-body.card-content-overflow]#pass:q[Contracts account for the interfaces allowing processes and UI to interact]#]
```

File: modules/ROOT/pages/getting-started-index.adoc [hunk 12]
```diff
@@ -1,57 +1,60 @@
...
+xref:bonita-studio-download-installation.adoc[[.card-title]#Bonita Studio Installation# [.card-body.card-content-overflow]#pass:q[Bonita's principal tool to create the main elements of all your projects]#]
 --
 
+[.card-section]
+== The 9-step hands-on journey to your first Bonita Living Application
+
 [.card.card-index]
 --
-xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[TODO]#]
+xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation, "most of the time" ;), starts with a process]#]
```
Comment (alachambre) [inline modules/ROOT/pages/getting-started-index.adoc]: it feels weird to have an ascii smiley here. Ascii smiley have to be put at the end of a sentence, else it's complicated to read. 
Moreover, I think that _"Most of the time" ;)_ is inadequate here. Keep the sentence simple, it's a card.

```suggestion
xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation usually starts with a process]#]
```

**Review by alachambre — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/getting-started-index.adoc (hunk 12 above)
Comment (NathalieC) [reply modules/ROOT/pages/getting-started-index.adoc]: They always start with a process. So, staying very plain, this is what I will say.

Comment (NathalieC) [inline modules/ROOT/pages/getting-started-index.adoc]:
```suggestion
xref:draw-bpmn-diagram.adoc[[.card-title]#Draw a BPMN diagram# [.card-body.card-content-overflow]#pass:q[Process automation always starts with a process]#]
```

**Review by alachambre — APPROVED** [review]

**Pull request merged**